tests/test_utils.py

```diff
@@ -41,6 +41,27 @@ def test_aioxmpp(self):
             "https://zombofant.net/xmlns/aioxmpp#internal"
         )
 
+    def test_inconsistent_rebinding_errors(self):
```

Comment:
Those tests should be on a separate TestCase subclass for the ``Namespaces`` class. A separate test here would check that the utils.namespaces object is an instance of ``Namespaces``.